app/views/components/_filter_section.html.erb

```diff
@@ -0,0 +1,33 @@
+<% add_app_component_stylesheet("filter-section") %>
+<%
+  raise ArgumentError, "heading_text is required" unless local_assigns[:heading_text]
+  raise ArgumentError, "heading_level for a section should not be heading level 1" if local_assigns[:heading_level] == 1
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+  open ||= nil
+  status_text ||= ""
+  heading_text ||= ""
+  heading_level ||= "2"
```

Comment:
This should be `2` (integer) for consistency with similar components

Reply:
good spot